utilities/upload_julia.sh

```diff
@@ -15,6 +15,9 @@ buildkite-agent artifact download "${UPLOAD_FILENAME}.tar.gz" .
 
 # These are the extensions that we will always upload
 UPLOAD_EXTENSIONS=( "tar.gz" )
+# Directory containing this script (relative path from current working directory)
+# Note: When used in contexts where absolute paths are required (e.g., for external tools),
+# prepend with $(pwd) to create an absolute path
 THIS_DIR="$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
Just use `realpath` to always make this an abspath, I don't think there's a reason not to.